content/recipes/inject.mdx

```diff
@@ -5,19 +5,19 @@ menu: Recipes
 
 # Store injecting
 
-The term of injection comes from `mobx-react` library. It is a legacy way of accessing the MobX stores across component tree. It was made because React [legacy context][context-legacy] was rather awkward to use.
+The store injection pattern was popularized by the `mobx-react` library. It refers to a now obsolete way of accessing the MobX stores across the component tree. It was introduced was because the React [legacy context](https://reactjs.org/docs/legacy-context.html) was rather awkward to use.
```

Comment:
```suggestion
The store injection pattern was popularized by the `mobx-react` library. It refers to a now obsolete way of accessing the MobX stores across the component tree. It was introduced because the React [legacy context](https://reactjs.org/docs/legacy-context.html) was rather awkward to use.
```